doc/md/guides/clickhouse/getting-started.mdx

```diff
@@ -0,0 +1,536 @@
+
+```sql
+CREATE TABLE `users` (
+  `id` UInt64,
+  `name` String NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE = MergeTree() ORDER BY id;
+```
+
+The `atlas schema inspect` command supports reading the database description provided by a [URL](/concepts/url) and outputting it in
+different formats, including [Atlas DDL](/atlas-schema/hcl.mdx) (default), SQL, and JSON. In this guide, we will
+demonstrate the flow using both the Atlas DDL and SQL formats, as the JSON format is often used for processing the
+output using `jq`.
+
+<Tabs>
+<TabItem value="hcl" label="Atlas DDL (HCL)" default>
+
+To inspect our locally-running ClickHouse instance, use the `-u` flag and write the output to a file named `schema.hcl`:
+
+```shell
+atlas schema inspect -u "clickhouse://localhost:9000/demo" > schema.hcl
+```
+
+Open the `schema.hcl` file to view the Atlas schema that describes our database.
+
+```hcl title="schema.hcl"
+table "users" {
+  schema = schema.demo
+  engine = MergeTree
+  column "id" {
+    null = false
+    type = UInt64
+  }
+  column "name" {
+    null = false
+    type = String
+  }
+  primary_key {
+    columns = [column.id]
+  }
+}
+schema "demo" {
+  engine  = sql("Memory")
+}
+```
+
+This block represents a [table](/atlas-schema/hcl.mdx#table) resource with `id`, and `name`
+columns. The `schema` field references the `demo` schema that is defined elsewhere
+in this document. In addition, the `primary_key` sub-block defines the `id` column as
+the primary key for the table. Atlas strives to mimic the syntax of the database that the user is working against. In
+this case, the type for the `id` column is `UInt64`, and `String` for the `name` column.
+
+</TabItem>
+<TabItem value="sql" label="SQL">
+
+To inspect our locally-running ClickHouse instance, use the `-u` flag and write the output to a file named `schema.sql`:
+
+```shell
+atlas schema inspect -u "clickhouse://localhost:9000/demo" --format '{{ sql . }}' > schema.sql
+```
+
+Open the `schema.sql` file to view the inspected SQL schema that describes our database.
+
+```sql title="schema.sql"
+CREATE TABLE `users` (
+    `id` UInt64,
+    `name` String)
+ ENGINE = MergeTree
+ PRIMARY KEY (`id`)
+ SETTINGS index_granularity = 8192;
+```
+
+</TabItem>
+</Tabs>
+
+:::info
+For in-depth details on the `atlas schema inspect` command, covering aspects like inspecting specific schemas,
+handling multiple schemas concurrently, excluding tables, and more, refer to our documentation
+[here](/declarative/inspect).
+:::
+
+To generate an Entity Relationship Diagram (ERD), or a visual representation of our schema, we can add the `-w` flag
+to the inspect command:
+
+```shell
+atlas schema inspect -u "clickhouse://localhost:9000/example" -w
+```
+
+![clickhouse-inspect](https://atlasgo.io/uploads/clickhouse/images/clickhouse-inspect.png)
+
+## Declarative Migrations
+The declarative approach allows the user to define the desired end schema, and Atlas provides a safe way to
```

Comment:
```suggestion
## Declarative Migrations

The declarative approach allows the user to define the desired end schema, and Atlas provides a safe way to
```

Reply:
The declarative approach allows users to manage their database schemas by simply..